src/client.js

```diff
@@ -871,6 +871,18 @@ client.prototype.handleMessage = function handleMessage(message) {
               { plan, planName },
               userstate,
             );
+          } else if (msgid === 'raid') {
+            const raider =
+              message.tags['msg-param-displayName'] ||
+              message.tags['msg-param-login'];
+            const viewers = Number(message.tags['msg-param-viewerCount']);
+            const userstate = message.tags;
+
+            if (userstate) {
+              userstate['message-type'] = 'raid';
+            }
+
+            this.emit('raid', channel, raider, viewers, userstate);
```

Comment:
Can you emit an object instead and update the documentation?

`this.emit('raid', { channel, raider, viewers, userstate });`

Reply:
Sure thing! Just curious, is there any reason why we'd emit an object here instead of individual variables like the other events?